test/clj/game_test/cards/assets.clj

```diff
@@ -79,6 +79,42 @@
       (card-ability state :corp alix 0)
       (is (= 8 (get-in @state [:corp :credit]))))) "Gain 4 credits from Alix")
 
+(deftest amani-senai
+  ;; Team Sponsorship - Install from HQ
```

Comment:
Wat :wink:

Reply:
Haha, oops. :) You'll never guess which test I modified for this one.